src/cli/cmds/synth.ts

```diff
@@ -74,5 +130,79 @@ async function fetchValidations(): Promise<ValidationConfig[] | undefined> {
   }
 }
 
+async function createHelmScaffolding(apiVersion: string, chartVersion: string, outdir: string) {
+  const tempHelmStructure = createFolderStructure();
+
+  const substituteValues = {
+    apiVersion: apiVersion,
+    version: chartVersion,
+    app: path.basename(path.resolve()),
+  };
+
+  try {
+    await sscaff(tempHelmStructure, outdir, substituteValues);
```

Comment:
Does `scaff` also cleanup its source directory? i.e `tempHelmStructure`? If not - we should delete it after its done.